devel/localhost-to-dockerhub

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Push the nextstrain/base and nextstrain/base-builder images from the local registry to Docker Hub.
+#
+# This pushes just the provided tag. If "latest" is provided as a second argument,
+# the provided tag will also be pushed as "latest"
+#
+set -euo pipefail
+
+if [[ $# -lt 1 ]]; then
+    echo "Please provide a tag." >&2
+    exit 1
+fi
+
+tag=$1
+
+BASE_IMAGE="nextstrain/base"
+BASE_BUILDER_IMAGE="nextstrain/base-builder"
+
+# copy local $tag to remote $tag
+skopeo copy --multi-arch=all --src-tls-verify=false docker://localhost:5000/$BASE_IMAGE:$tag docker://docker.io/$BASE_IMAGE:$tag
+skopeo copy --multi-arch=all --src-tls-verify=false docker://localhost:5000/$BASE_BUILDER_IMAGE:$tag docker://docker.io/$BASE_BUILDER_IMAGE:$tag
+
+if [ $# -eq 2 ] && [$2 == "latest" ]; then
+    # copy local $tag to remote latest
```

Comment:
Probably also condition that `$1 != "latest"` so it isn't copied twice (probably wouldn't hurt, but also not necessary).

Reply:
Could be reduced to `[[ "$tag" != latest && "${2:-}" == latest ]]`.

Better to use `[[ … ]]` expressions, regardless.

Reply:
Nice, updated: 59c99bf3a49275548a40bc5aa5df0f92e5686a8c